tinygrad/tensor.py

```diff
@@ -2753,6 +2753,31 @@ def celu(self, alpha=1.0):
     """
     return self.maximum(0) + (alpha * ((self / alpha).exp() - 1)).minimum(0)
 
+  def prelu(self, weight:Tensor, channel_dim:int|None=None):
```

Comment:
don't quite understand the constraint in onnx spec and tests, but i'd prefer this api to match torch without indirection, then somehow call this in onnx prelu (we have other functions that works that way because api is close but not identical anyway). let me know what you think